libr/core/cmd_search_rop.c

```diff
@@ -565,6 +565,147 @@ static char* rop_classify_arithmetic (RCore *core, RList *ropList) {
 	return arithmetic;
 }
 
+static char* rop_classify_arithmetic_const (RCore *core, RList *ropList) {
+	char *esil_str, *op, *constant;
+	char *arithmetic = NULL, *esil_flg = NULL, *esil_main = NULL;
+	RListIter *iter_src1, *iter_r, *iter_dst, *iter_ops, *iter_const;
+	RRegItem *item_src1, *item_dst;
+	RList *head, *constants;
+	RList *ops_list = NULL, *flg_read = NULL, *flg_write = NULL, *reg_read = NULL,
+		*reg_write = NULL, *mem_read = NULL, *mem_write = NULL;
+	const bool romem = r_config_get_i (core->config, "esil.romem");
+	const bool stats = r_config_get_i (core->config, "esil.stats");
+	ut64 *op_result = malloc (sizeof (*op_result));
```

Comment:
do you really need to allocate this in the heap? why not just use R_NEW0(ut64) if not